Jellyfin.Api/Helpers/AudioHelper.cs

```diff
@@ -154,4 +160,61 @@ public async Task<ActionResult> GetAudioStream(
             transcodingJobType,
             cancellationTokenSource).ConfigureAwait(false);
     }
+
+    /// <summary>
+    /// Gets audio data to draw a waveform bar.
+    /// </summary>
+    /// <param name="itemId">The item id.</param>
+    /// <returns>A <see cref="FileStream"/> containing the resulting .dat file.</returns>
+    public async Task<FileStream> GetAudioWaveForm(Guid itemId)
+    {
+        var item = _libraryManager.GetItemById<BaseItem>(itemId)
+            ?? throw new ResourceNotFoundException();
+
+        var libraryOptions = BaseItem.LibraryManager.GetLibraryOptions(item);
+        var saveInMediaFolder = libraryOptions.SaveLocalMetadata;
+        var saveFileName = Path.GetFileNameWithoutExtension(item.Path) + ".csv";
+        string outputPathCsv = saveInMediaFolder
+            ? Path.Combine(item.ContainingFolderPath, saveFileName)
+            : Path.Combine(item.GetInternalMetadataPath(), saveFileName);
+        string outputPathJson = outputPathCsv.Replace("csv", "json", StringComparison.Ordinal);
+
+        if (!File.Exists(outputPathJson))
+        {
+            var processStartInfo = new ProcessStartInfo("ffprobe", $"-v error -select_streams a:0 -show_entries stream=sample_rate -of csv=p=0 \"{item.Path}\"") { RedirectStandardOutput = true, UseShellExecute = false };
+            int sampleRate = 0;
+            int fileSampleRate = item.GetMediaStreams()[0].SampleRate ?? 44100;
+            // This value sets the length of a frame, the amount of measurements per second.
+            int samplesPerSecond = 2;
+            sampleRate = Math.Max(fileSampleRate / samplesPerSecond, 1);
+
+            // escape double quotes and backslashes
+            string itemPathEscaped = item.Path.Replace("\\", "\\\\", StringComparison.Ordinal).Replace("\"", "\\\"", StringComparison.Ordinal);
```

Comment:
I'm not sure if this escape is valid under windows where backslashes have its own use. Wouldn't this just escaping each and every valid windows path?